apps/condo/migrations/20220721114406-0151_contactrolehistoryrecord_contacthistoryrecord_role_and_more.js

```diff
@@ -0,0 +1,78 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Create model contactrolehistoryrecord
+--
+CREATE TABLE "ContactRoleHistoryRecord" ("organization" uuid NULL, "name" text NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "createdBy" uuid NULL, "updatedBy" uuid NULL, "deletedAt" timestamp with time zone NULL, "newId" jsonb NULL, "dv" integer NULL, "sender" jsonb NULL, "history_date" timestamp with time zone NOT NULL, "history_action" varchar(50) NOT NULL, "history_id" uuid NOT NULL);
+--
+-- Add field role to contacthistoryrecord
+--
+ALTER TABLE "ContactHistoryRecord" ADD COLUMN "role" uuid NULL;
+--
+-- Add field canManageContactRoles to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canManageContactRoles" boolean DEFAULT false NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canManageContactRoles" DROP DEFAULT;
```

Comment:
You need to specify which of the default roles can manage contact roles, add something like this:
`UPDATE "OrganizationEmployeeRole" SET "canManageContactRoles" = true WHERE "name" = ANY(ARRAY['employee.role.Dispatcher.name', 'employee.role.Administrator.name']);`

Reply:
Done